src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -133,6 +139,7 @@ function MultiCombobox<Item>({
   disabled = false,
   placeholder = '',
   itemToString = item => String(item),
+  itemToGroup,
```

Comment:
IMO there should be a default value here, showcasing how to make items not be grouped at all

```suggestion
  itemToGroup = () => null,
```